db/schema.rb

```diff
@@ -10,11 +10,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_04_23_115256) do
+ActiveRecord::Schema.define(version: 2020_04_25_064420) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "messages", force: :cascade do |t|
+    t.string "name"
+    t.integer "age"
```

Comment:
ロールバックした上で，マイグレーションファイルを修正し，マイグレーションを行えば，こちらは自動的に修正されるでしょう。

```suggestion
    t.string "age"
```

Reply:
ありがとうございます。
同手順でマイグレーションしたのですが、db/schema.rbは変化しませんでした。
また、マイグレーションを修正した後、手動でdb/schema.rbを修正したのですが、
db/schema.rbのt.string "content"が勝手に元に戻ってしまいます。